Contact form

stino-x opened the pull request:
this commits adds in a form and form section collecting basic data from its recipients

stino-x committed:
branch creation

just added a few things to make this branch consistent with the previous

ClintonYounge committed:
Update style.css

stino-x committed:
adjusting my form

tweaking the arrangement of my css

Jrmcuervo committed:
Added input and placeholders

stino-x committed:
Updating READ-ME file

i updated the list of contributors to this branch

stino-x committed:
Form spree

updated the form spree link

stino-x committed:
resolving stylelint issues

i corrected stuff the stylelint pointed out

huseyinbiyik commented on index.html:
```diff
@@ -164,6 +164,30 @@ <h3>Skills</h3>
+            <h1 class="get-started">Get Started</h1>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus architecto, atque laudantium autem vel explicabo soluta quos fugit quod distinctio. Ea ullam ut facere numquam minima dignissimos? Mollitia, esse iure.</p>
+        <form action="https://formspree.io/f/xoqzvkpg" method="post">
+            <label for="Name"><input type="text" placeholder="Your name" maxlength="30" class="name"></label>
+            <label for="Email"><input type="email" placeholder="Your email" class="email"></label>
+            <label for="Message"><textarea type="text" name="" maxlength="500" class="message" placeholder="Enter your message here.."></textarea></label>
```
Good job! But let's add `required` to all of our input and textarea tag to complete our project requirements:
<img width="546" alt="Screenshot 2023-01-19 at 8 27 41 PM" src="https://user-images.githubusercontent.com/25887253/213517510-e797a105-5a62-4b17-9c11-2067b20f4709.png">

If you try to submit a form, you will have the same warnings from formspree.

huseyinbiyik commented on index.html:
```diff
@@ -164,6 +164,30 @@ <h3>Skills</h3>
+    <section class="section-4">
+        <div class="mobile-form-wrapper">
+            <h1 class="get-started">Get Started</h1>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus architecto, atque laudantium autem vel explicabo soluta quos fugit quod distinctio. Ea ullam ut facere numquam minima dignissimos? Mollitia, esse iure.</p>
```
Please make sure you have padding on the right side of the paragraph. Check the differences below.
<img width="931" alt="Screenshot 2023-01-19 at 8 34 11 PM" src="https://user-images.githubusercontent.com/25887253/213519008-e0141741-cca6-4ba2-b231-cce774efe695.png">
<img width="322" alt="Screenshot 2023-01-19 at 8 34 55 PM" src="https://user-images.githubusercontent.com/25887253/213519015-bc721c5b-7655-469f-b26c-95b6e7f96c59.png">

Good job with the font-family but let's fix the font-size to match the Figma template.
<img width="1010" alt="Screenshot 2023-01-19 at 8 36 44 PM" src="https://user-images.githubusercontent.com/25887253/213519325-41a94a30-3fdf-40f1-876d-6a0653ded1a1.png">


<img width="814" alt="Screenshot 2023-01-19 at 8 36 33 PM" src="https://user-images.githubusercontent.com/25887253/213519321-4cea8d63-2f46-4982-b4a7-cfca9914e51b.png">

stino-x replied:
@huseyinbiyik i think its the max-width i set wrong not that it needs padding thanks for the suggestion though

stino-x replied:
@huseyinbiyik please can you check again, i dont think the font size is different from the figma file, please confirm this

huseyinbiyik replied:
Hey @stino-x ,
They are different ↓
<img width="472" alt="Screenshot 2023-01-19 at 10 34 22 PM" src="https://user-images.githubusercontent.com/25887253/213542757-ca24409c-e44b-4b53-bc54-42749c469076.png">
<img width="576" alt="Screenshot 2023-01-19 at 10 35 04 PM" src="https://user-images.githubusercontent.com/25887253/213542761-36fb3eaa-e088-430f-ae92-ad0098486089.png">

stino-x replied:
@huseyinbiyik so just to confirm i set it to 14px just like you said but it looks very different from what is in the figma file visually, am i still supposed to change it?

huseyinbiyik reviewed (changes requested):
Hi @stino-x, @ClintonYounge and @Jrmcuervo,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![Almost there](https://media.giphy.com/media/Rv6GGaNrRbtNConlYo/giphy.gif)

## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁

## Required Changes ♻️
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"

_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

stino-x committed:
Implementing requested changes

add the required attribute to form
changed the font size of the paragraph in the form section
aligned the input area in the form

huseyinbiyik commented on index.html:
```diff
@@ -164,6 +164,30 @@ <h3>Skills</h3>
+            <h3 class="get-started">Get Started</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus architecto, atque laudantium autem vel explicabo soluta quos fugit quod distinctio. Ea ullam ut facere numquam minima dignissimos? Mollitia, esse iure.</p>
+        <form action="https://formspree.io/f/xoqzvkpg" method="post">
+            <label for="Name"><input type="text" placeholder="Your name" maxlength="30" class="name" required></label>
+            <label for="Email"><input type="email" placeholder="Your email" class="email"required></label>
+            <label for="Message"><textarea type="text" name="" maxlength="500" class="message" placeholder="Enter your message here.." required></textarea></label>
```
As I mentioned earlier, please fix input left side padding like on the Figma template. It needs to be 16px.
<img width="499" alt="Screenshot 2023-01-19 at 11 18 02 PM" src="https://user-images.githubusercontent.com/25887253/213551620-ad7fa178-656a-436c-9c9e-24f175cc4687.png">
<img width="554" alt="Screenshot 2023-01-19 at 11 19 02 PM" src="https://user-images.githubusercontent.com/25887253/213551623-04752acf-57ae-40d7-90fd-c593fc9c7ec0.png">

stino-x replied:
@huseyinbiyik hello again do i have to implement the input font as well shown in the figma file?

huseyinbiyik replied:
Yes, yours is 13.33px, the template has 14px. You can fix it. 👍

stino-x replied:
@huseyinbiyik please could you suggest how to do this ? i know its not a change you suggested but id like to implement it for knowledge purposes

huseyinbiyik commented on index.html:
```diff
@@ -164,6 +164,30 @@ <h3>Skills</h3>
+        <form action="https://formspree.io/f/xoqzvkpg" method="post">
+            <label for="Name"><input type="text" placeholder="Your name" maxlength="30" class="name" required></label>
+            <label for="Email"><input type="email" placeholder="Your email" class="email"required></label>
+            <label for="Message"><textarea type="text" name="" maxlength="500" class="message" placeholder="Enter your message here.." required></textarea></label>
```
Please make sure you added name attribute correctly. So needs to be "message" here. Otherwise your contact form will not submit the data.

huseyinbiyik reviewed (changes requested):
Hi @stino-x,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![tenor](https://user-images.githubusercontent.com/25887253/205415161-00e79d1b-af68-4f10-9b2c-265f460dc5da.gif)

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

stino-x committed:
implementing further required changes

 added padded to input area
added font style and size to inputs

leonardodiasb reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @stino-x 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Linter set up well and checks pass 🟢
 - Github flow followed 👌

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

stino-x committed:
New images

i added in new images to use in the next branch

stino-x committed:
form edit

ezekielgn committed:
Add form in the HTML file

bobb-Rob reviewed (approved):
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @stino-x , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://camo.githubusercontent.com/2c67944c14f5c74b9b607df18d73efb0c423dd148b3e10cb75b445a00220332e/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f5061536e555a573145657970756c734f77642f67697068792e676966)

## Highlights
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
